src/Microsoft.OData.Core/UriParser/Parsers/ODataPathParser.cs

```diff
@@ -910,6 +933,90 @@ private void CreateFirstSegment(string segmentText)
             this.CreateDynamicPathSegment(null, identifier, parenthesisExpression);
         }
 
+        /// <summary>Creates the first <see cref="ODataPathSegment"/> for a request.</summary>
+        /// <param name="segmentText">The text of the segment.</param>
+        private bool CreateSegmentForEscapeFunction(string segmentText)

```

Comment:
>CreateSegmentForEscapeFunction [](start = 21, length = 30)

This is not really creating a segment for an escape function, right? It's creating a segment that an escape function may be bound to?

Reply:
Correct. 

---
In reply to: [369792392](https://github.com/OData/odata.net/pull/1621#discussion_r369792392) [](ancestors = 369792392)